features/storage/FEATURE_STORAGE/cfstore/source/cfstore_svm.cpp

```diff
@@ -39,7 +44,7 @@ static ARM_DRIVER_STORAGE *cfstore_svm_storage_drv = &ARM_Driver_Storage_MTD_K64
 #endif /* CFSTORE_CONFIG_BACKEND_FLASH_ENABLED */
 
 /* the storage volume manager instance used to generate virtual mtd descriptors */
-static StorageVolumeManager volumeManager;
+StorageVolumeManager volumeManager;
```

Comment:
Ideally, the StorageVolumeManager should be a singleton for each flash region. It is not and people can use it wrong. Getters would be preferable, but reasons.